client/me/help/help-contact/index.jsx

```diff
@@ -418,12 +421,18 @@ const HelpContact = React.createClass( {
 	 */
 	getView: function() {
 		const { olark, confirmation, sitesInitialized, isSubmitting } = this.state;
+		const { isTicketSupportConfigurationReady, isTicketSupportEligible, ticketSupportRequestError } = this.props;
+
 		const showHappychatVariation = this.shouldUseHappychat();
 		const showChatVariation = olark.isUserEligible && olark.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( olark.details.isConversing || olark.isUserEligible );
+		const showKayakoVariation = ! showChatVariation && ( olark.details.isConversing || isTicketSupportEligible );
```

Comment:
Would it be worth extracting `showKayakoVariation` into a selector?

Maybe later even something like:

```js
function getEligibleSupportLevel( state ) {
  if ( ... ) {
    return HELP_LEVEL_CHAT;
  } else if ( ... ) {
    return HELP_LEVEL_TICKET;
  } else {
    return HELP_LEVEL_FORUMS;
  }
}
```